persistence-model/src/main/resources/application-dev.conf

```diff
@@ -4,13 +4,26 @@ akka {
     warn-about-java-serializer-usage = on
     serializers {
       institution = "hmda.persistence.serialization.institutions.InstitutionProtobufSerializer"
+      submission = "hmda.persistence.serialization.submission.SubmissionProtobufSerializer"
       filing = "hmda.persistence.serialization.filing.FilingProtobufSerializer"
+      parser = "hmda.persistence.serialization.parser.HmdaFileParserProtobufSerializer"
+      larValidated = "hmda.persistence.serialization.validation.LarValidatedProtobufSerializer"
+      upload = "hmda.persistence.serialization.upload.UploadProtobufSerializer"
     }
     serialization-bindings {
       "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionCreated" = institution
       "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionModified" = institution
       "hmda.persistence.messages.events.institutions.FilingEvents$FilingCreated" = filing
       "hmda.persistence.messages.events.institutions.FilingEvents$FilingStatusUpdated" = filing
+      "hmda.persistence.messages.events.institutions.SubmissionEvents$SubmissionCreated" = submission
```

Comment:
I wasn't sure about this. I updated `application-dev.conf` to have the same serialization config as `application.conf`. Is that the right way to do this?